hw2/03-count.js

```diff
@@ -1,3 +1,15 @@
-// Add your code here
+const input = document.querySelector("input");
+var originalText = document.getElementById("text").innerHTML;
 
-input.addEventListener('keydown', handleKeyDown);
+// Used this as a resource: https://dev.to/kokaneka/highlight-searched-text-on-a-page-with-just-javascript-17b3#:~:text=The%20'mark'%20tag&text=If%20you%20surround%20any%20text,in%20this%20striking%20yellow%20color.&text=That%20makes%20highlighting%20searched%20text%20quite%20a%20simple%20task%20then.
+function handleKeyUp() {
+  let inputStr = input.value.trim();
+  if (inputStr !== "") {
+    let text = originalText;
+    let regex = new RegExp(inputStr, "g");
```

Comment:
While your code works, one small change that I can mention here is that I noticed that it highlights all the words that contain the input word, even if it's a part of a larger word. For example, when searching for '**in**', words like '**in**credulity' and 'spr**in**g' are also highlighted, which might not be the desired behavior as per the screenshots provided in HW2. Could you please take a look and make some modifications to ensure that only the exact input word is highlighted?

I am also attaching the screenshot from our HW2 for reference.

![image](https://user-images.githubusercontent.com/114516950/234771206-49dfc50e-24da-4f4a-a789-97f6af7ecd77.png)